packages/atomic/cypress/integration/result-list/folded-result-list.cypress.ts

```diff
@@ -0,0 +1,128 @@
+import {generateComponentHTML, TestFixture} from '../../fixtures/test-fixture';
+import {
+  addFoldedResultList,
+  buildTemplateWithoutSections,
+} from './result-list-actions';
+import {makeChildComponents} from './folded-result-list-utils';
+import {
+  FoldedResultListSelectors,
+  getAtomicResultChildren,
+  getAtomicChildResultAtIndex,
+  getAtomicGrandChildResultAtIndex,
+} from './folded-result-list-selectors';
+
+const setSource = () => {
+  cy.intercept({method: 'POST', path: '**/rest/search/v2**'}, (request) => {
+    request.body.aq = '@source==("iNaturalistTaxons")';
+  });
+};
+
+describe('Folded Result List Component', () => {
+  it('should show child results', () => {
+    new TestFixture()
+      .with(setSource)
+      .with(
+        addFoldedResultList(
+          buildTemplateWithoutSections([
+            generateComponentHTML('atomic-result-link'),
+            makeChildComponents(),
+          ])
+        )
+      )
+      .init();
+
+    FoldedResultListSelectors.firstResult()
+      .contains('Pyrenomycetes')
+      .should('be.visible');
+
+    getAtomicChildResultAtIndex(0)
+      .find('atomic-result-link')
+      .contains('Stubble Lichens');
+    getAtomicChildResultAtIndex(1)
+      .find('atomic-result-link')
+      .contains('Opercs');
+  });
+
+  it('should show grandchild results', () => {
+    new TestFixture()
+      .with(setSource)
+      .with(
+        addFoldedResultList(
+          buildTemplateWithoutSections([
+            generateComponentHTML('atomic-result-link'),
+            makeChildComponents(makeChildComponents()),
+          ])
+        )
+      )
+      .init();
+
+    FoldedResultListSelectors.firstResult()
+      .contains('Pyrenomycetes')
+      .should('be.visible');
+
+    getAtomicChildResultAtIndex(0)
+      .find('atomic-result-link')
```

Comment:
could be replaced with `find(resultLinkComponent)` (defined in `result-link-selectors.ts`)